apps/condo/domains/resident/schema/DiscoverServiceConsumersService.js

```diff
@@ -36,43 +47,98 @@ const DiscoverServiceConsumersService = new GQLCustomSchema('DiscoverServiceCons
                 const { data } = args
                 const { address, unitName, unitType, billingAccount, resident, dv, sender } = data
 
-                const residents = await Resident.getAll(context,
-                    resident ? { id: resident.id, deletedAt: null } : {
-                        address: address,
-                        unitName: unitName,
-                        unitType: unitType,
-                        deletedAt: null,
-                    })
-
-                const billingAccounts = await BillingAccount.getAll(context,
-                    billingAccount ? { id: billingAccount.id, deletedAt: null } : {
-                        context: { status: 'Finished', deletedAt: null },
-                        property: { address: address, deletedAt: null  },
-                        unitName: unitName,
-                        unitType: unitType,
+                const reqId = get(context, ['req', 'id'])
+
+                const billingAccounts = await BillingAccount.getAll(
```

Comment:
By using server schema you need to control returned fields in `gql.js`. So `find`, `getOne` and `getByCondition` is safer imho... 

(Optional notice btw)

Reply:
I'd like to load all relations (context, organization, property)